CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 ### Added
 
 - Added the normalization parameters of ```torchvision.transforms.Normalize``` as ```transform_kwargs``` in the ```ImageClassificationInputTransform``` ([#1178](https://github.com/PyTorchLightning/lightning-flash/pull/1178))
+- Added `BASE_MODEL_NAME` key in `VISSLAdapter`. ([#1184](https://github.com/PyTorchLightning/lightning-flash/pull/1184))
```

Comment:
Should this instead be a `Fixed` rather than added?

Reply:
I was also thinking about this, but it adds a key, and then it also fixes a bug. Uhmm, what do you prefer?

Reply:
I would go fixing :smiley: I try to use changelog as user facing changes (e.g. in this case "fixes support for dino in the image embedder" or something like that).

Reply:
Makes sense, thanks @ethanwharris - I'll update this with the commit including tests once ready.